framework/include/eventloop/eventloop.h

```diff
@@ -44,8 +44,39 @@ enum el_result_error_e {
 	EVENTLOOP_OUT_OF_MEMORY = -3,
 	EVENTLOOP_LOOP_FAIL = -4,
 	EVENTLOOP_NOT_FINISHED = -5,
+	EVENTLOOP_INVALID_HANDLE = -6,
 };
 
+/**
+ * @brief Pre defined Event list
+ * @details These values can be used for event type of event loop.\n
+ * If user wants to use new events, those should be defined ahead.\n
+ * An example of a pre-defined events are shown below.\n
+ * \n
+ * enum el_defined_event_e {\n
```

Comment:
Why not event_type_e?
You use ```int type``` at API. It is difficult to know this is a type of event which is used in API as type argument.